test/framework/config.py

```diff
@@ -681,8 +682,29 @@ def test_update_build_option(self):
         """Test updating of a build option."""
         self.assertEqual(build_option('banned_linked_shared_libs'), None)
 
-        update_build_option('banned_linked_shared_libs', '/usr/lib64/libssl.so.1.1')
+        old_val = update_build_option('banned_linked_shared_libs', '/usr/lib64/libssl.so.1.1')
         self.assertEqual(build_option('banned_linked_shared_libs'), '/usr/lib64/libssl.so.1.1')
+        self.assertEqual(old_val, None)
```

Comment:
Maybe extend this a bit to also update a build option that has a very specific default value, like:

```python
orig_cleanup_builddir = update_build_option('cleanup_builddir', False)
self.assertFalse(build_option('cleanup_builddir'))
self.assertTrue(orig_cleanup_builddir)

orig_pr_target_account = update_build_option('pr_target_account', 'test_pr_target_account')
self.assertEqual(build_option('pr_target_account'), 'test_pr_target_account')
self.assertEqual(orig_pr_target_account, 'easybuilders')
```

Because `None` is what you'd also get if there's no `return` at all...

Reply:
Done!